src/View/Helper/FormHelper.php

```diff
@@ -102,6 +109,11 @@ public function __construct(View $View, array $config = [])
 
         $this->_defaultWidgets = $this->_widgets + $this->_defaultWidgets;
 
+        $this->_controlMethod = 'input';
```

Comment:
For me it's more about the logical "thats the status quo" and "thats the old bc fallback" kind of way. the time speed is irrelevant anyway, as you well know :)